internal/store/fleet.go

```diff
@@ -26,7 +26,9 @@ type Fleet interface {
 	UpdateStatus(ctx context.Context, orgId uuid.UUID, fleet *api.Fleet) (*api.Fleet, error)
 	UpdateStatusMultiple(ctx context.Context, orgId uuid.UUID, fleets ...*api.Fleet) error
 	DeleteAll(ctx context.Context, orgId uuid.UUID) error
-	Delete(ctx context.Context, orgId uuid.UUID, name string) error
+	Delete(ctx context.Context, orgId uuid.UUID, names ...string) error
+	UnsetOwnerForMatchingFleets(ctx context.Context, tx *gorm.DB, orgId uuid.UUID, owner string) error
```

Comment:
I am surprised that we need to pass gorm into the store. The reason for the store is to abstract away the underlying ORM.

Reply:
Could we wrap this in a Type maybe something like.

type Transaction(func(tx *gorm.DB))

then interface can be

```
UnsetOwnerForMatchingFleets(ctx context.Context, Transaction, orgId uuid.UUID, owner string) error
```